coriolis/osmorphing/osmount/base.py

```diff
@@ -124,27 +123,13 @@ def _get_vgnames(self):
 
     def _get_lv_paths(self):
         """ Returns list with paths of available LVM volumes. """
+        out = self._exec_cmd("sudo lvdisplay -c").decode().strip().split('\n')
+        LOG.debug("Decoded `lvdisplay` output data: %s", out)
         lvm_paths = []
-        out = json.loads(
-            self._exec_cmd(
-                "sudo lvs -o lv_path,lv_name --report-format json").decode())
-        LOG.debug("Decoded `lvs` output data: %s", out)
-
-        reports = out.get("report", [])
-        if not reports:
-            return []
-
-        if len(reports) > 1:
-            LOG.warn("Multiple LVM reports found: %s", reports)
-        report = reports[0]
-
-        for i, lvm in enumerate(report.get("lv", [])):
-            lvm_name = lvm.get("lv_name", str(i))
-            lvm_path = lvm.get("lv_path")
-            if not lvm_path:
-                LOG.warn("No path for lvm volume '%s'", lvm_name)
-            else:
-                lvm_paths.append(lvm_path)
+        for line in out:
+            lvm_vol = line.strip().split(':')
+            if lvm_vol is not None:
```

Comment:
List emptiness checks should be just `if list`.
`lvm_vol is not None` will only return True if `lvm_vol = None`, and not if the `lvm_vol = []` so we're still open to the `IndexError` below